cluster/gce/gci/configure-helper.sh

```diff
@@ -2059,7 +2079,7 @@ function start-kube-controller-manager {
     params+=" --concurrent-service-syncs=${CONCURRENT_SERVICE_SYNCS}"
   fi
   if [[ "${NETWORK_PROVIDER:-}" == "kubenet" ]]; then
-    params+=" --allocate-node-cidrs=true"
```

Comment:
Specifying the `true` is not acceptable. (Will cause kube-controller-manager container unable to start).